### ISSUE TITLE
Does anyone have a simple DocVQA workflow?

### ISSUE THREAD

**Issue by tiago-falves** [kijai/ComfyUI-Florence2#80]
![image](https://github.com/user-attachments/assets/585fa218-9880-4fea-9f41-55621f8f91e4)
I am not being able to have a description of the product (Not of the whole image)
Does anyone have a simple DocVQA workflow?
Thank you!

**Comment by 4lt3r3go** [comment]
yeah how to use this? i tryed everything

**Comment by botslop** [comment]
I don't believe Florence-2-large will work with DocVQA (unless you read something that specified that). Have you tried using the other model mentioned in the README?

https://huggingface.co/HuggingFaceM4/Florence-2-DocVQA
